apps/dao-producer-job/src/main.ts

```diff
@@ -0,0 +1,54 @@
+import { Haus } from '@daohaus/dao-data';
+import { Keychain } from '@daohaus/common-utilities';
+import * as Queue from 'bull';
+
+const SLEEP_TIME = parseInt(process.env.SLEEP_TIME) || 300;
+const REDIS_HOST = process.env.SLEEP_TIME || 'redis://127.0.0.1:6379';
+
+// TODO: Commented out networks are invalid
+const networks = [
+  // '0x1',
+  '0x5',
+  // '0x64',
+  // '0xa',
+  // '0x89',
+  // '0xa4b1',
+  // '0xa4ec',
+] as Array<keyof Keychain>;
+
+// sleep time expects milliseconds
+function sleep(time) {
+  return new Promise((resolve) => setTimeout(resolve, time));
+}
+
+const main = async () => {
+  console.log('Start processing daos');
+  const producerQueue = new Queue('dao-producer', REDIS_HOST);
+  try {
+    const haus = Haus.create();
+    for (const networkId of networks) {
+      console.log(`Starting network ${networkId.toString()}`);
+      const resp = await haus.query.listDaos({ networkId });
```

Comment:
This will work well for now - but we'll need to deal with pagination at some point.

basicall the resp will give you a new prop called nextPaging that you'll pass on a subsequent listDaos call until nextPaging is undefined.

```
haus.listDaos({netoworkId, paging: resp.nextPaging})
```

Reply:
Good call

Reply:
Updated pagination